pkgs/applications/audio/zrythm/default.nix

```diff
@@ -122,36 +139,50 @@ stdenv.mkDerivation rec {
     rtaudio
     rtmidi
     rubberband
+    sassc
```

Comment:
Isn't sassc a css compiler that is run on the host?

Reply:
no I think this is about why it is in `buildInputs` and not in `nativeBuildInputs`